src/components/pages/PageCreateCards/PageCreateCards.jsx

```diff
@@ -0,0 +1,140 @@
+import React from 'react';
+import { useSelector } from 'react-redux';
+import TemplateDefault from '../../Templates/TemplateDefault';
+import firebase from '../../../firebase/firebase';
+
+function PageCreateCards() {
+  const [cardName, setCardName] = React.useState('');
+  const [sideOne, setSideOne] = React.useState('');
```

Comment:
Please import `useState` off of React so you don't have to repeat `React.useState` constantly.